docker-shell-helper.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+#####################################
+#     Docker Helper Script Usage    #
```

Comment:
I haven't looked too hard at what this is doing but I'm sure it could be useful to others.

Worth adding to the README.md and also to the commit message i this is being included please.

Reply:
Yeagh, can add to the README.md.

See my comment above too. Just not sure in the value of actually making this change. Of course totally happy to work on valuable stuff... but part of the pod is about working out what is and is not anymore, so there will be the usual differences in opinion!